app/(Components)/(Form)/Inputs/InputSection.tsx

```diff
@@ -16,10 +17,18 @@ export const InputSection: React.FC<{InputList: object}> = ({InputList}) => {
 					case 'time':
 						InputComponent = InputTime;
 						break;
+					case 'remoteOnly':
+						InputComponent = Checkbox;
+						break;
 					default:
 						InputComponent = Input;
 						break;
 				}
+
+				const handleRemoteOnlyEvent = (checked: boolean) => {
+					input.setData(!checked);
+				};
```

Comment:
I don't think that specific behavior must be in this component

Reply:
pending for SVG

Reply:
Je suis d'accord que ça devrait pas être là, et je pense que l'on est tombé dans un cas où à vouloir trop faire de refacto et éviter la duplication on tombe sur des composants qui sont impossible à clean, il faudra faire gaffe pour svg, là on va pas perdre de temps là dessus 😊